files/puppet_helper.groovy

```diff
@@ -680,6 +680,21 @@ class Actions {
         ]
         break
 
+      //  active directory
+      case 'hudson.plugins.active_directory.ActiveDirectorySecurityRealm':
+        config = [
+          setSecurityRealm: [
+            (className): [
```

Comment:
@danielpalstra @jhoblitt If I'm reading this correctly the arguments here need to be updated? @danielpalstra are you using latest version of the active_directory plugin?
I need to set site as blank and there should be a realm.server parameter?